packages/ens-utils/src/ensname.ts

```diff
@@ -326,6 +326,19 @@ export function getRegistrationPotential(name: ENSName): RegistrationPotential {
   }
 }
 
+/**
+ * Splits a label into its individual characters.
+ * 
+ * This function uses the spread operator to split the string into an array of its Unicode characters (UTF-32) compatible with ens-normalize.
```

Comment:
A few thoughts on this line:

1. Best not to document implementation details. Ex: "This function uses the spread operator to split the string into an array..."
2. Is there a special reason to identify these are UTF-32 characters? Perhaps there's an important idea there? Is this different from the regular / default representation of characters in JavaScript? Or is that splitting different than splitting of UTF-16 characters?
3. Can you help me understand the significance of saying that this is "compatible with ens-normalize"? I imagine there's an important idea there, but need your help to understand it 😄

Reply:
Thank you. I have changed documentation to:
In JavaScript, the ".split()" method of a string may give different results  because it returns an array of UTF-16 code units.
By "compatible with ens-normalize" I meant it uses the same implementation but maybe it is not important.